### ISSUE TITLE
Palpable great auricular nerve

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7668]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
601596 
PMID: 14574644  1/18

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
greater auricular nerve thickening  added
